src/sql/workbench/parts/dashboard/browser/widgets/insights/insightsWidget.contribution.ts

```diff
@@ -44,3 +46,8 @@ ExtensionsRegistry.registerExtensionPoint<IInsightTypeContrib | IInsightTypeCont
 		}
 	}
 });
+
+CommandsRegistry.registerCommand('azdata.widget.setAutoRefreshState',
+	function (accessor: ServicesAccessor, widgetId: string, connectionId: string, autoRefresh: boolean) {
+		setWidgetAutoRefreshState(widgetId, connectionId, autoRefresh);
```

Comment:
Should probably be doing some validation here since calls can pass in anything - the typing isn't going to prevent that.